cmd/snap-bootstrap/partition/sfdisk.go

```diff
@@ -100,10 +100,7 @@ func (sf *SFDisk) Layout() (*gadget.LaidOutVolume, error) {
 
 // Create creates the partitions listed in positionedVolume
 func (sf *SFDisk) Create(pv *gadget.LaidOutVolume) ([]DeviceStructure, error) {
-	// Layout() will update sf.partitionTable
```

Comment:
We could either return an error, making it very explicit, or call Layout() when the partition table is nil so it would be populated. What do you think?

Reply:
Thank you, I like calling layout if the partition table is nil.